### ISSUE TITLE
Votul electronic nu functioneaza

### ISSUE THREAD

**Issue by mihai-dragan** [evot-usr/proiect_legislativ#1]
Oricine are dreptul sa voteze trebuie sa aiba posibilitatea sa voteze, dar votul electronic NU este o solutie:
https://www.youtube.com/watch?v=w3_0x6oaDmI

**Comment by mv0** [comment]
Nu are nicio legatura cu proiectul. Nu esti singurul care da cel mai prost extemplu posibil. As zice sa te intorci la scoala. Frauda si tentativa vine din alte alte parti: infrastructura si unde se tin rezultate si nu procesul descris aici. Votul electronic nu numai ca este o solutie, dar este cea mai buna solutie pe care am putea sa o adoptam ca natie.

**Comment by mihai-dragan** [comment]
Corect, atacul la persoana este cel mai bun argument, am incredere acum ca stiti ce faceti. Multumesc pentru sfatul cu intoarcerea la scoala.

**Comment by mv0** [comment]
De asemenea votul nu este un drept fundamental. Sunt destule exemple unde drepturile civice sunt private in momentul in care esti la puscarie. Argumentul ca uite ce zice un gigel pe internet da dovada ca ai grave carente si nici macar  nu ti-ai dat interesul sa citesti textul, sa studiezi problema sa vii cu argumente tangibile de ce votul electronic nu este o solutie.

**Comment by mihai-dragan** [comment]
Daca nu intelegi ca atacul la persoana nu isi are locul intr-o discutie civilizata, nu are rost sa mai stam de vorba aici. Mult noroc in continuare.

**Comment by mv0** [comment]
Atacurile sunt mai la tine in cap. Inainte sa vezi razboaie imaginare as zice sa studiezi problema si apoi sa vii discutam. Frumos ar fi sa inchizi thread-ul asta.

**Comment by alexvoda** [comment]
Atitudiena de aici nu este incurajatoare dar voi aduce totusi argumente:

> 
> 
> De asemenea votul nu este un drept fundamental. 

Articolul 21.3 din declaratia drepturilor omului:
> (3) The will of the people shall be the basis of the authority of government; this will shall be expressed in periodic and genuine elections which shall be by universal and equal suffrage and shall be held by secret vote or by equivalent free voting procedures.
https://www.un.org/en/universal-declaration-human-rights/index.html

Articolul 10 din aceasta propunere:
> Sistemul de vot electronic permite cetățeanului să verifice independent:
> a) dacă votul său a fost recepționat;
> b) dacă votul său a fost înscris corect;
> c) dacă votul său a fost numărat așa cum a fost înscris.

Verificarea ulterioara si secretul votului sunt mutual incompatibile.

**Comment by alexvoda** [comment]
> As zice sa te intorci la scoala. 

>  ai grave carente si nici macar nu ti-ai dat interesul

Acestea sunt atacuri la persoana.

**Comment by mv0** [comment]
> Atitudiena de aici nu este incurajatoare dar voi aduce totusi argumente:
> 
> > De asemenea votul nu este un drept fundamental.
> 
> Articolul 21.3 din declaratia drepturilor omului:
> 
> > (3) The will of the people shall be the basis of the authority of government; this will shall be expressed in periodic and genuine elections which shall be by universal and equal suffrage and shall be held by secret vote or by equivalent free voting procedures.
> > https://www.un.org/en/universal-declaration-human-rights/index.html
> 
> Articolul 10 din aceasta propunere:
> 
> > Sistemul de vot electronic permite cetățeanului să verifice independent:
> > a) dacă votul său a fost recepționat;
> > b) dacă votul său a fost înscris corect;
> > c) dacă votul său a fost numărat așa cum a fost înscris.
> 
> Verificarea ulterioara si secretul votului sunt mutual incompatibile.

Nu vad de ce ar fi incompatibile, daca anonimitatea nu poate fi pastrata si este imperativa (eu personal nu o vad ca o problema, dar altii poate se simt lezati de ce au votat -- si asta nu garaneaza ce ceilalti pot vedea fix cu cine ai votat) sunt sigur ca existe metode prin care asta poate asigurata. Cu priviere la ONU, un text de la 1948, in total context in care ne aflam astazi, de o institutie ce promoveaza activ migratia ilegala nu are are absolut nici un fel de fundament in secolul 21.

**Comment by Andrei-Paul** [comment]
> Verificarea ulterioara si secretul votului sunt mutual incompatibile.

Fals. Poti atasa votului un "mesaj" pe care doar tu il poti decripta si sa verifici ca in continuare corespunde, ulterior numararii. Pentru public "mesajul" ala e gunoi si nu poate fi legat de o persoana anume decat in cazuri extreme (1 om a fost unicul votant la o sectie de votare) pentru care poate fi depistat si cu metoda actuala de vot, si asta doar in caz de terminal static dintr-o sectie de votare, nu terminal personal legat la o random retea dintr-o random cafenea.

**Comment by mv0** [comment]
> > As zice sa te intorci la scoala.
> 
> > ai grave carente si nici macar nu ti-ai dat interesul
> 
> Acestea sunt atacuri la persoana.

@alexvoda, @mihai-dragan Imi cer scuze in mod public daca te-am atacat/ofensat. Scopul a fost sa opresc valul de TomScott pe care le-am observat de-a lungul timpului si care este dat ca un exemplu foarte prost din punctul meu de vedere.

**Comment by alexvoda** [comment]
> Scopul a fost sa opresc valul de TomScott pe care le-am observat de-a lungul timpului si care este dat ca un exemplu foarte prost din punctul meu de vedere.

Acel video este ce a mai concisa cale de explica cuiva argumentele impotriva votului electronic. Popularitatea videului este de inteles si binemeritata.

Votul este important sa fie anonim, secret si voluntar pentru a nu fi impusa alegerea de catre alt cineva. Votul pe hartie este secret pentru ca:
1. Se presupune ca observatorii care au interese divergente se asigura ca primesti un buletin de vot curat si intact
2. Esti singur in cabina de vot
3. Se presupune ca observatorii care au interese divergente se asigura ca nu te obliga nimeni sa arati buletinul de vot dupa ce ai iesit din cabina
4. Votul odata introdus in urna se amesteca cu restul voturilor de la acea sectie si devine imposibil de identificat cui apartine. (Cu cat e numarul mai mare de alegatori la acea sectie cu atat e mai dificil de identificat cui apartine un vot. Daca esti singurul, evident ca poate fi identificat cui apartine)

> Nu vad de ce ar fi incompatibile, daca anonimitatea nu poate fi pastrata si este imperativa (eu personal nu o vad ca o problema, dar altii poate se simt lezati de ce au votat -- si asta nu garaneaza ce ceilalti pot vedea fix cu cine ai votat) sunt sigur ca existe metode prin care asta poate asigurata. 

> Fals. Poti atasa votului un "mesaj" pe care doar tu il poti decripta si sa verifici ca in continuare corespunde, ulterior numararii

Daca tu poti verifica rezultatul votului atunci altcineva te poate obliga sa verifici si sa arati rezultatul.
Daca votezi de la distanta(adica in lipsa intimitatii cabinei de vot) sau daca poti verifica votul ulterior, atunci seful/cineva din familie/membrul de partid/oricine are autoritate asupra ta te poate obliga sa dezvalui votul si te poate penaliza daca nu ai votat corespunzator.

E o diferenta intre a da mita electorala si sa speri ca alegatori voteaza ce le-ai zis, si a conditiona primirea mitei de verificarea rezultatului. Cineva care primeste mita are totusi libertatea in cabina de vot sa puna stampila pe ce doreste atata timp cat puntele 1, 2 si 3 de mai sunt respectate. Cineva care primeste buletinul de vot gata completat are libertatea in cabina de vot sa invalideze votul.


Nu exista metode prin care sa permiti cuiva sa verifice propriul vot dar sa interzici altcuiva sa oblige pe altii sa dezvaluie votul. Nu este o problema de natura tehnica. xkcd relevant: https://xkcd.com/538/
 
Am abordat aici numai problama pastrarii anonimitatii. Nu intru in problemele tehnice legate de posibilitatiile de fraudare. Video-ul lui Tom Scott pune mai mult accentul pe problemele tehnice.

> Cu priviere la ONU, un text de la 1948, in total context in care ne aflam astazi, de o institutie ce promoveaza activ migratia ilegala nu are are absolut nici un fel de fundament in secolul 21.
Definitia data de ONU este cel mai apropape lucru care il avem ca definitie a termenului "drept fundamental (al omului)". Propui o definitie alternativa?

**Comment by mv0** [comment]
> Acel video este ce a mai concisa cale de explica cuiva argumentele impotriva votului electronic. Popularitatea videului este de inteles si binemeritata.

Nu am nimic impotriva unei critice, dar critica lui Tom Scott se refera la total altceva decat la ce se refera textul acesta. Tom Scott vorbeste preponderent de masini de vot electronic https://en.wikipedia.org/wiki/Voting_machine, iar aici in text vorbim de votat electronic folosind ceva asemantor cu https://en.wikipedia.org/wiki/Electronic_identification. Daca vrem sa discutam care sunt potentialele probleme ar trebuie sa ne uitam la criticile aduse la sistemul din Estonia si la ce am zis mai sus: infrastructura si stocarea informatiei care poate fi manipulata de actori malitiosi, de unde banuiesc ca se trage si marea frica a populatiei in ne-cunostinta de cauza. Dar inca o data, textul de aici nu se refera la asta ci mai degraba la procesul efectiv de votare. 

> Daca tu poti verifica rezultatul votului atunci altcineva te poate obliga sa verifici si sa arati rezultatul.
Daca votezi de la distanta(adica in lipsa intimitatii cabinei de vot) sau daca poti verifica votul ulterior, atunci seful/cineva din familie/membrul de partid/oricine are autoritate asupra ta te poate obliga sa dezvalui votul si te poate penaliza daca nu ai votat corespunzator.

Asta e problema sociala, si pe strada, un infractor ma poate pandii la colt de strada cand scot bani de la ATM si sa ma forteze sa-i dau card si pinul sau sa-mi iau banii direct din portofel. Premiza ca cineva te constrange mi se pare o tema falsa si nu are legatura cu textul in cauza. Da, sunt constient ca poate fi o problema, dar este o problema ce trebuie adresata de alte institutii ale statului insarcinate cu altfel de probleme.  

> Am abordat aici numai problama pastrarii anonimitatii. Nu intru in problemele tehnice legate de posibilitatiile de fraudare.

E o problema falsa asta cu anonimitatea.

**Comment by alexvoda** [comment]
In acelasi timp, pentru orice vot PUBLIC (anonimitatea nu este necesara, sau este chiar de nedorit), la scara mare, votul electronic este perfect.

Votul reprezentantiilor de exemplu este un vot public. Ai interesul sa stii ca cei pe care i-ai ales sa te reprezinte, voteaza in interesul tau.

**Comment by alexvoda** [comment]
@mv0 Problema anonimitatii nu este o problema falsa si este aplicabila atat masinilor de vot cat si votului online.
Prefer sa stiu ca e imposibil pt angajator sa zica ca ma concediaza daca nu votez in interesul lui.
Prefer sa stiu ca e imposibil pt un membru de familie sa zica ca ma dezmosteneste daca nu votez cum doreste.
Prefer sa stiu ca e imposibil pt oricine are autoritate asupra mea sa verifice cum am votat.

Prefer ca fata de toti acestia sa pot minti in legatura cu cine am votat si sa fie imposibil sa ma verifice sau sa ma oblige sa dezvalui.

Ti-ar convenii cand esti la liceu sa iti zica mama sau tata sa votezi cu cine zic ei ca altfel nu primesti bani de facultate?
Ce alta institutie a statului adreseaza o astfel de problema?

De asemenea, prefer sa fie imposibil pt orice entitate sa verifice ulterior cu cine am votat.
Prefer ca un guvern viitor sa nu poata sa verifice cu cine am votat cu 20 de ani in urma si sa devin victima a politiei politice. 
Prefer sa, nu ma bazez ca asocierea intre token-uri si identitate este stearsa.
Prefer sa, nu am incredere in calculatorul care genereaza tokenurile, pentru ca nu pot sa verific ca ruleaza codul publicat. Si prefer sa nu am incredere in cel care auditeaza si spune ca e ok, pentru ca acela poate fi mituit.

Acestea toate sunt probleme ale votului electronic si nu sunt probleme tehnice. 
Votul electronic face aceste probleme foarte usor de intampinat.
Pentru a avea aceste probleme la votul pe hartie trebuie convinsi oameni cu interese divergente.
Atata timp cat punctele 1-4 sunt respectate, acestea sunt probleme inexistente. Ai libertatea sa minti cat vrei si sa votezi cum vrei.

Cum ai spus, acestea sunt probleme sociale. Prefer sa nu creez niste probleme sociale noi.

**Comment by alexvoda** [comment]
Am mai citit un pic.

> Art.11. - Sistemul de vot electronic stochează doar datele cu caracter personal ale alegătorului necesare desfășurării procesului electoral electronic și numai pe perioada strict necesară atingerii acestui scop, până la expirarea termenelor pentru contestarea rezultatului alegerilor.

> Art.12. - Sistemul de vot electronic și orice parte autorizată trebuie să protejeze datele de autentificare ale alegătorului, astfel încât părțile neautorizate să nu poată utiliza, intercepta, modifica sau dobândi cunoștințe despre aceste date.

> Art. 44. - Constituie contravenţii, dacă nu sunt săvârşite în astfel de condiţii încât să constituie potrivit legii penale infracţiuni, următoarele fapte:
> a) încălcarea prevederilor art. 11 și ale art. 12;

> Art. 45. - (1) Contravenţiile prevăzute la art. 44 se sancționează după cum urmează:
> a) faptele prevăzute la lit. a) și la lit. b) cu amendă de la 15.000 lei la 30.000 lei;

Pe bune, 30,000 RON pentru o baza de date cu viitoarele tinte pt politia politica? Un CHILIPIR! Daca un partid ar intentiona sa preia puterea si sa instaureze dictatura, o astfel de suma pentru a cunoaste identitatea inamicilor e maruntis.

Pt comparatie, legislatia GDPR prevede:
> Up to €10 million, or 2% annual global turnover – whichever is higher; or
> Up to €20 million, or 4% annual global turnover – whichever is higher.

**Comment by senciucserban** [comment]
> Pe bune, 30,000 RON pentru o baza de date cu viitoarele tinte pt politia politica? Un CHILIPIR! Daca un partid ar intentiona sa preia puterea si sa instaureze dictatura, o astfel de suma pentru a cunoaste identitatea inamicilor e maruntis.

Plus ca este o **contaventie** si nu o fapta penala.

**Comment by alexvoda** [comment]
> 
> 
> > Pe bune, 30,000 RON pentru o baza de date cu viitoarele tinte pt politia politica? Un CHILIPIR! Daca un partid ar intentiona sa preia puterea si sa instaureze dictatura, o astfel de suma pentru a cunoaste identitatea inamicilor e maruntis.
> 
> Plus ca este o **contaventie** si nu o fapta penala.

Nu tocmai

> "dacă nu sunt săvârşite în astfel de condiţii încât să constituie potrivit legii penale infracţiuni"

Dar cu siguranta exista o modalitate de a masca o scurgere de informatii ca o contraventie in loc de infractiune.

**Comment by mv0** [comment]
Asa e, contraventiile/pedepsele sunt foarte mici, de atlfel cam cum e toata legistalatia penala, asta trebuie vorbit/discutat cu cei care au redactat textul. 

@alexvoda iti inteleg temerile si unele sunt chiar reale, altele m-as teme mai mult pentru viata mea decat cum functioneaza un sistem de vot electronic, pentru ca descrii un regim totalitar.... Poate cei care au redactat textul arunca un ochi aici si incearca sa adreseze intr-o oarecare masura si problema anonimitatii.

**Comment by alexvoda** [comment]
> 
> m-as teme mai mult pentru viata mea decat cum functioneaza un sistem de vot electronic, pentru ca descrii un regim totalitar....

Exact, legile intr-o democratie trebuie sa apere impotriva potentialei germinari a semintelor de dictatura.

**Comment by xandrw** [comment]
Daca nu functioneaza, parerea mea este sa vii cu argumente proprii, nu cu video de pe youtube.
Alt fel, inseamna ca accepti sa stai la cozi interminabile si chiar sa nu votezi datorita acestor cozi, cum s-a intamplat pe 26 mai 2019.

**Comment by iuliux** [comment]
Nu vreti sa abordati punctual in issue-uri separate fiecare punct? Asa e greu de dezbatut si concluzionat, intr-un mega thread "Totul e gresit".
